zoltar-tests/src/test/java/com/spotify/zoltar/ModelLoaderTest.java

```diff
@@ -52,35 +54,27 @@ public void close() throws Exception {
   }
 
   @Test
-  public void preload() {
+  public void preload() throws InterruptedException, ExecutionException, TimeoutException {
     final ModelLoader<DummyModel> loader = ModelLoader
-        .lift(DummyModel::new)
-        .with(Preloader.preload());
-
-    assertThat(loader.get().toCompletableFuture().isDone(), is(true));
-  }
-
-  @Test
-  public void preloadTimeout() {
-    final ModelLoader<DummyModel> loader = ModelLoader
-        .lift(() -> {
-          Thread.sleep(Duration.ofSeconds(10).toMillis());
+        .load(() -> {
+          Thread.sleep(Duration.ofMillis(500).toMillis());
```

Comment:
Any reason this shouldn't be something much smaller? Like `5ms`?

Reply:
yeah 😄